Sources/Data/OAMapStyleSettings.mm

```diff
@@ -59,6 +60,9 @@ @interface OAMapStyleSettings ()
 @end
 
 @implementation OAMapStyleSettings
+{
+    OAAppSettings* _settings;
```

Comment:
OAMapStyleSettings should not read OAMapStyleSettings. Looks like this class needs only method to save few params.